src/locale/locales/hi/messages.po

```diff
@@ -77,112 +77,112 @@ msgstr ""
 #: src/components/ProfileHoverCard/index.web.tsx:398
 #: src/screens/Profile/Header/Metrics.tsx:23
 msgid "{0, plural, one {follower} other {followers}}"
-msgstr ""
+msgstr "0, plural, one {फ़ॉलोअर} other {फ़ॉलोअर}}"
 
 #: src/components/ProfileHoverCard/index.web.tsx:402
 #: src/screens/Profile/Header/Metrics.tsx:27
 msgid "{0, plural, one {following} other {following}}"
-msgstr ""
+msgstr "{0, plural, one {फ़ॉलोइंग} other {फ़ॉलोइंग}}"
 
 #: src/view/com/util/post-ctrls/PostCtrls.tsx:312
 msgid "{0, plural, one {Like (# like)} other {Like (# likes)}}"
-msgstr ""
+msgstr "{0, plural, other {पसंद करें (# पसंद)}}"
```

Comment:
This isn't a strict preference but not sure if we should say पसंद over लाइक (transliteration of like) since many people usually say that when referring to liking in social medias

Reply:
I was torn between पसंद and लाइक. I chose पसंद because that's what both Youtube and Twitter's Hindi UI's use.

Reply:
that's fair, I was not aware of that